airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflow.java

```diff
@@ -63,12 +63,6 @@
   @SignalMethod
   void retryFailedActivity();
 
-  /**
-   * Use for testing in order to simulate a job failure.
-   */
-  @SignalMethod
-  void simulateFailure();
-
```

Comment:
This was a bad design that I introduce and revisited here because I need to have a failing workflow. It was a signal to override the syncOutputStatus return by the child workflow and considered it as failed.

It got replaced by a custom implementation if the SyncWorkflow in the test folder that return the expected syncStatusOutput.